src/useWorker.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-console */
```

Comment:
```suggestion
/* eslint-disable no-underscore-dangle, no-console */
```

Or we can disable these rules globally?

Reply:
Yes after merging this PR, I want to open a PR removing these comments, and rearranging the package.json (remove babel6, prettier etc.) 👍